ansible/roles/tenzir/tenzir-node.service

```diff
@@ -28,6 +28,9 @@ CapabilityBoundingSet=
 AmbientCapabilities=
 RestrictSUIDSGID=yes
 
+# limits
+LimitNOFILE=65535
```

Comment:
I considered going all the way to `LimitNOFILE=infinity`, but in the case of a file descriptor leak it seems better to have the node fail at some point rather than the system hitting the maximum number of global open files.

Reply:
Agreed. Could you expand on the comment a bit here to inform users about this decision?

Reply:
I think it's the wrong audience; for the end users it doesn't matter *why* we set the limit this way. I'll copy the comment in the commit description instead.